.github/workflows/create_feedstocks.yml

```diff
@@ -26,6 +26,15 @@ jobs:
           poll-interval-seconds: 2
           github-token: ${{ secrets.GITHUB_TOKEN }}
         continue-on-error: true
+        
+      - name: commit any changes upon checkout
+        run: |
+          git add *
+          git commit --allow-empty -am "make sure we have no windows line endings"
```

Comment:
Why allow empty?

Reply:
Otherwise, the commit will fail if things are fine and I didn't want to have to handle that.